lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -192,19 +192,24 @@ def set_top_view(self):
 
     def _init_axis(self):
         '''Init 3D axes; overrides creation of regular X/Y axes'''
-        self.w_xaxis = axis3d.XAxis('x', self.xy_viewLim.intervalx,
+        self.xaxis = axis3d.XAxis('x', self.xy_viewLim.intervalx,
                                     self.xy_dataLim.intervalx, self)
-        self.xaxis = self.w_xaxis
-        self.w_yaxis = axis3d.YAxis('y', self.xy_viewLim.intervaly,
+        self.yaxis = axis3d.YAxis('y', self.xy_viewLim.intervaly,
                                     self.xy_dataLim.intervaly, self)
-        self.yaxis = self.w_yaxis
-        self.w_zaxis = axis3d.ZAxis('z', self.zz_viewLim.intervalx,
+        self.zaxis = axis3d.ZAxis('z', self.zz_viewLim.intervalx,
                                     self.zz_dataLim.intervalx, self)
-        self.zaxis = self.w_zaxis
+        # Provide old aliases
+        self.w_xaxis = self.xaxis
+        self.w_yaxis = self.yaxis
+        self.w_zaxis = self.zaxis
 
         for ax in self.xaxis, self.yaxis, self.zaxis:
             ax.init3d()
 
+    def get_zaxis(self):
+        '''Return the :class:`ZAxis<~.axis3d.Axis>` instance.'''
```

Comment:
Does not link in the docs.

Reply:
Yes. Any idea why?

Reply:
For web links there must be a space between the display text and `<`. Maybe it's that.

I don't know if this format is supported by (class)-roles (never seen this use-case before).

Reply:
I see why. `axis3d.Axis` does not have any docstring. I guess I will add one.

Reply:
I think ``:py:meth:`Queue.get <Queue.Queue.get>` `` worked for me at some point. At least, I've written that in https://github.com/sphinx-doc/sphinx/issues/4416.

Try ``:class:`ZAxis <mpl_toolkits.mplot3d.axis3d.Axis>` `` or ``Z\ `.Axis` ``. In the latter case, be sure to make the docstring a raw string.